### PR TITLE
Add two troubleshootings about installation

### PR DESCRIPTION
**Pull request by peter279k**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/operations/troubleshooting.md [hunk 1]
```diff
@@ -17,6 +17,37 @@ title: Troubleshooting
...
+### You Cannot Update Deb Packages from ClickHouse Repository with Apt-get {#you-cannot-update-deb-packages-from-clickhouse-repository-with-apt-get}
+
+- The issue may be happened when the GPG key is changed.
+
+Please use the following scripts to resolve the issue:
+
+```bash
+sudo apt-key del "3A9E A119 3A97 B548 BE14  57D4 8919 F6BD 2B48 D754"
+sudo rm /etc/apt/sources.list.d/clickhouse.list
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754
```
Comment (Felixoid) [inline docs/en/operations/troubleshooting.md:29]: It's completely redundant. Just repeating the command `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754` updates the key

**Review by Felixoid — CHANGES_REQUESTED** [review]
One point is too long, it should be short as `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754`

I'd propose moving it to install and using a `spoiler` (details?) in the https://clickhouse.com/docs/en/install/#install-from-deb-packages

Another point: the PR description is wrong, see the [template](https://github.com/ClickHouse/ClickHouse/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

File: docs/en/operations/troubleshooting.md (hunk 1 above)
Comment (peter279k) [reply docs/en/operations/troubleshooting.md]: Just simplify the GPG key updating approach.

File: docs/en/operations/troubleshooting.md [hunk 2]
```diff
@@ -17,6 +17,34 @@ title: Troubleshooting
...
+N: Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'https://packages.clickhouse.com/deb stable InRelease' doesn't support architecture 'i386'
+```
+
+To resolve the above issue, please use the following script:
+
+```bash
+sudo rm /var/lib/apt/lists/packages.clickhouse.com_*
+sudo apt-get clean
+sudo apt-get autoclean
+sudo rm /var/lib/dpkg/arch
```
Comment (Felixoid) [inline docs/en/operations/troubleshooting.md:45]: It looks a bit more elegant

```suggestion
sudo rm /var/lib/apt/lists/packages.clickhouse.com_* /var/lib/dpkg/arch
sudo apt-get clean
sudo apt-get autoclean
```

**Review by Felixoid — APPROVED** [review]

**Pull request merged**